client/main.lua

```diff
@@ -196,36 +196,32 @@ end
 local function openVehCatsMenu(category, targetVehicle)
     local vehMenu = {}
 
-    for k, v in pairs(VEHICLES) do
-        if VEHICLES[k].category == category then
-            if not config.vehicles[k] then
-                lib.print.debug('Vehicle not found in config.vehicles. Skipping: '..k)
-            elseif type(config.vehicles[k].shop) == 'table' then
-                for _, shop in pairs(config.vehicles[k].shop) do
-                    if shop == insideShop then
-                        vehMenu[#vehMenu + 1] = {
-                            title = ('%s %s'):format(v.brand, v.name),
-                            description = locale('menus.veh_price')..lib.math.groupdigits(v.price),
-                            serverEvent = 'qbx_vehicleshop:server:swapVehicle',
-                            args = {
-                                toVehicle = v.model,
-                                targetVehicle = targetVehicle,
-                                closestShop = insideShop
-                            }
-                        }
+    local function insertVehicle(data)
```

Comment:
avoid functions inside of functions

Reply:
any reason why it bad?

Reply:
It can make code confusing. The preferred alternative is to move both functions to a separate file, with the inside function instead being a local function to that file to reduce it's scope.